cms/models/permissionmodels.py

```diff
@@ -4,7 +4,11 @@
 from django.contrib.auth.models import Group, UserManager
 from django.contrib.sites.models import Site
 from django.core.exceptions import ImproperlyConfigured
-from django.utils import importlib
+try:
```

Comment:
This import should be moved to line 20 (after `if DJANGO_1_6:`).

And maybe, when we're at it, we should use `from importlib import import_module` instead, and `... = import_module(...)` on line 23 to make things shorter to read.